app/src/main/res/layout/welcome.xml

```diff
@@ -34,7 +34,7 @@
         android:layout_marginEnd="@dimen/home_welcome_text_view_margin_end"
```

Comment:
In `all_topics.xml` remove this `component_color_home_activity_select_a_topic_to_start_text_color` and `color_palette_select_a_topic_to_start_text_color` and 
use the `component_color_shared_primary_text_color`

Reply:
Done.